README.rst

```diff
@@ -335,6 +336,30 @@ The following options can be only specified in the task decorator:
   has set up ``BATCH_QUEUES`` for the specific queue (see the *Configuration*
   section).
 
+- ``schedule``
+
+  If given, makes a task execute periodically. Pass either:
+
+  - a function that takes the current datetime as an argument.
+  - a tuple ``(f, args)``, where ``f`` takes the current datetime as the first
+    argument, followed by the additional args.
+
+  The schedule function must return the next task execution datetime, or
+  ``None`` to prevent periodic execution. The function is executed to determine
+  the initial taks execution date when a worker is initialized, and to determine
+  the next execution date when the task is about to get executed.
+
+  For most common scenarios, the ``periodic`` built-in function can be passed:
+
+  - ``periodic(seconds=0, minutes=0, hours=0, days=0, weeks=0, start_date=None,
```

Comment:
I can see that this is a possibility. But for now, I want to keep this simple, and if someone needs different schedules, they can write a custom scheduler function. If needs arise, we can add more complex schedulers at a later point in time.